NuGet.config

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+   <packageRestore>
+    <add key="enabled" value="True" />
+  </packageRestore>
+  <packageSources>
+    <add key="local" value="./src/package/bin/Debug" />
+  </packageSources>
+  <config>
+    <add key="globalPackagesFolder" value="packages" />
```

Comment:
Is there a reason we need to have a separate directory?

We could save a lot of space by using system nuget package store.

Reply:
I did this to restore `Xunit logger 1.0.0-dev`. Don't want to restore stale 1.0.0-dev.